code/mob/living.dm

```diff
@@ -1502,10 +1504,33 @@ var/global/icon/human_static_base_idiocy_bullshit_crap = icon('icons/mob/human.d
 
 	if (pushpull_multiplier != 0) // if we're not completely ignoring pushing/pulling
 		if (src.pulling)
-			. *= (pull_speed_modifier(move_target) * pushpull_multiplier)
+			if (istype(src.pulling, /atom/movable) && !(src.is_hulk() || (src.bioHolder && src.bioHolder.HasEffect("strong"))))
```

Comment:
Bringing this in to here for performance (i.e. only used once, so might as well have here)?

Reply:
That and it lets me use the values derived from movement modifiers (such as mob_pull_multiplier)